common/config/jest/jest.config.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
   // Work around mapping ESM issues; see https://github.com/Azure/azure-sdk-for-js/issues/34195#issuecomment-2850957045
   moduleNameMapper: {
     '^@typespec/ts-http-runtime/internal/(.*)$':
-      '<rootDir>/../../../common/temp/node_modules/.pnpm/@typespec+ts-http-runtime@0.2.2/node_modules/@typespec/ts-http-runtime/dist/commonjs/$1/internal.js'
+      '<rootDir>/../../../common/temp/node_modules/.pnpm/@typespec+ts-http-runtime@0.2.3/node_modules/@typespec/ts-http-runtime/dist/commonjs/$1/internal.js'
```

Comment:
Can we use regex here so we won't need to update the package version everytime? Something like:
```
'<rootDir>/../../../common/temp/node_modules/.pnpm/@typespec+ts-http-runtime@\\d+\\.\\d+\\.\\d+/node_modules/@typespec/ts-http-runtime/dist/commonjs/$1/internal.js'
```